railties/test/application/test_runner_test.rb

```diff
@@ -446,7 +446,7 @@ def test_pass_TEST_env_on_rake_test
       create_test_file :models, 'account'
       create_test_file :models, 'post', pass: false
 
-      output =  Dir.chdir(app_path) { `bin/rake test TEST=test/models/post_test.rb` }
+      output =  Dir.chdir(app_path) { `bin/rails test TEST=test/models/post_test.rb` }
```

Comment:
`TEST` is only for backwardscompatibility with `rake test` as `bin/rails test` already supports running tests from a single file more cleanly.